eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
+
+type RouteList []*Route
```

Comment:
it has to be public, as it's used in other packages. If you want to remove it, we can open an issue.